src/codecatalyst/auth.ts

```diff
@@ -37,6 +38,8 @@ export class CodeCatalystAuthStorage {
     }
 }
 
+export const onboardingUrl = vscode.Uri.parse('https://codecatalyst.aws/onboarding/view')
```

Comment:
Would https://codecatalyst.aws/explore be more generally applicable "landing page" for this purpose?